deploy/helm/charts/values.yaml

```diff
@@ -111,7 +111,7 @@ jivaCSIPlugin:
     repository: openebs/jiva-csi
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: 2.11.0
+    tag: 2.12.0
```

Comment:
```suggestion
    tag: 2.12.1
```

Reply:
can the repository name be changed to "openebs/jiva-operator" in the above lines..